Integrations/Integration-Salesforce.yml

```diff
@@ -0,0 +1,174 @@
+configuration:
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: Consumer Key
+  name: client_id
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Consumer Secret
+  name: client_secret
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |-
+    var authUrl = 'https://login.salesforce.com/services/oauth2/token';
+    var uriPrefix = '/services/data/v20.0/';
+
+    function getNewToken() {
+        var request = {
+                grant_type: 'password',
+                client_id: params.client_id,
+                client_secret: params.client_secret,
+                username: params.credentials.identifier,
+                password: params.credentials.password
+            };
+
+        var body = encodeToURLQuery(request).substr(1);
```

Comment:
Is that worth adding as a call inside common integration to be reused? not sure, do it if you think so also